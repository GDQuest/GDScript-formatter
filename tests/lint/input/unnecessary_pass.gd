#region Good

func good():
	pass

#endregion

#region Bad

func bad():
	print("Hello")
	pass

#endregion
