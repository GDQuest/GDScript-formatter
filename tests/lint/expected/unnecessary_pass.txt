tests/lint/input/unnecessary_pass.gd:12:unnecessary-pass:warning: Unnecessary 'pass' statement when other statements are present